Assigment1/CustomerRegistration/src/main/java/com/customer/helper/ConnectionProvider.java

```diff
@@ -0,0 +1,40 @@
+package com.customer.helper;
+
+import java.sql.Statement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+
+
+
+public class ConnectionProvider {
+    public static Connection con;
+    public static Statement st;
+
+    static{
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+            String url="jdbc:mysql://localhost:3307/hotwax";
+            String name="root";
+            String pass="@Rohit12";
```

Comment:
Please use Property files to save your credentials. It is a bad practice to use credentials in Java file.